assets/fighters/slinger/slinger.fighter.yaml

```diff
@@ -45,18 +45,19 @@ spritesheet:
       repeat: false
     attacking:
       frames: [14, 18]
-# this should be changed to a ranged attack
+
 attacks:
-  - name: "punch"
+  - name: "projectile"
     damage: 7
     frames:
-      startup: 1
-      active: 2
-      recovery: 3
+      startup: 4
+      active: 4
+      recovery: 4
```

Comment:
having all these frames be the same *might* cause issues but maybe not. Will look more closely when I get a chance.

Reply:
seems good the way its implemented, only the startup frame is ever being used :+1: